src/zarr/api/asynchronous.py

```diff
@@ -646,7 +646,11 @@ async def create(
     if zarr_format == 2 and chunks is None:
         chunks = shape
     if zarr_format == 3 and chunk_shape is None:
-        chunk_shape = shape
+        if chunks is not None:
+            chunk_shape = chunks
+            chunks = None
+        else:
+            chunk_shape = shape
```

Comment:
Is this actually a good fallback? Perhaps worth bringing in [`guess_chunks`](https://github.com/zarr-developers/zarr-python/blob/c677da4b0bc3e600b23cad0cea1253b7923810b0/src/zarr/v2/util.py#L102) from the v2 codebase (in a later PR).

Reply:
I agree - I was just changing one thing here.